app/src/sharedTest/java/org/oppia/app/profile/AddProfileChooserActivityTest.kt

```diff
@@ -59,7 +59,7 @@ import javax.inject.Singleton
 
 @RunWith(AndroidJUnit4::class)
 @LooperMode(LooperMode.Mode.PAUSED)
-class AddProfileActivityTest {
+class AddProfileChooserActivityTest {
```

Comment:
This is incorrect. We don't have to rename `AddProfileActivity` to `AddProfileChooserActivity`

Reply:
Please revert changes in this file which are related to `AddProfile`